asciidoc/courses/llm-fundamentals/modules/2-vectors-semantic-search/lessons/3-c-create-vector-index/lesson.adoc

```diff
@@ -0,0 +1,95 @@
+= Create a Vector Index
```

Comment:
I dont think its practical for learners to remember the instructions from the last lesson and then apply it in this lesson. There is too much information to retain. 

The collapsibles give the solution and I think in practice everyone would simply reveal the solution and run it.

Break this lesson into 3 discrete lessons and walking learners through the solution asking them to run, predict and investigate what has happened would support them in learning what has happened.